### PR TITLE
Build Statistic Reporting for TeamCity

### PR DESCRIPTION
**Pull request by michaelmyers**
This adds the flag `--teamcity` and then within the `simple_output` coverage service it leverages the [default build statistics for coverage](https://confluence.jetbrains.com/display/TCDL/Custom+Chart#CustomChart-listOfDefaultStatisticValues) to [report](https://confluence.jetbrains.com/display/TCDL/Build+Script+Interaction+with+TeamCity#BuildScriptInteractionwithTeamCity-ReportingBuildStatistics) the results to TeamCity, a continuous integration server by JetBrains.

Example usage would be:

```sh
$ slather coverage --teamcity -s
```

We also can consider cherry-picking [joshrlesch's commit](https://github.com/hudl/slather/commit/4a101fdd315f41aa4ad80289230d7ef04ffc2965) from [PR 132](https://github.com/SlatherOrg/slather/pull/132) which adds support for TeamCity within the `coverall` coverage service to complete the solution.   

This is my first time with ruby so I am looking forward to feedback.

### CODE REVIEW

**Comment by kdawgwilk** [comment]
:+1:

**Comment by choli** [comment]
this works, thanks :+1:

File: spec/slather/coverage_service/simple_output_spec.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
     proj = Slather::Project.open(FIXTURES_PROJECT_PATH)
     proj.build_directory = TEMP_DERIVED_DATA_PATH
     proj.input_format = "profdata"
-    proj.configure
+    proj.send(:configure)
```
Comment (neonichu) [inline spec/slather/coverage_service/simple_output_spec.rb]: This seems unrelated to the PR. Leftover or am I missing something?

**Comment by neonichu** [comment]
@michaelmyers Thanks a lot for the PR :+1: Would you mind adding a Changelog entry?

File: spec/slather/coverage_service/simple_output_spec.rb (hunk 1 above)
Comment (michaelmyers) [inline spec/slather/coverage_service/simple_output_spec.rb]: You are correct, this isn't needed.

**Comment by michaelmyers** [comment]
@neonichu removed unnecessary code and added a changelog entry.  Let me know if you need anything else.  Thanks for your work on the project!

**Comment by neonichu** [comment]
Thanks, it's good to go, will merge once Travis is :green_heart:

**Comment by shritekale** [comment]
Does anyone know when this would be released?

**Comment by michaelmyers** [comment]
This does not answer your question but if you want to use it now you can use:

```
gem 'slather', :git => 'git://github.com/SlatherOrg/slather', ref: 'f4d2e3b'
```

in your Gemfile.
